mne/channels/channels.py

```diff
@@ -279,7 +280,8 @@ def set_eeg_reference(self, ref_channels=None):
         mne.set_bipolar_reference
         """
         from ..io.reference import set_eeg_reference
-        return set_eeg_reference(self, ref_channels, copy=False)[0]
+        return set_eeg_reference(self, ref_channels, copy=False,
+                                 verbose=verbose)[0]
```

Comment:
you don't need to pass `verbose`  to the underlying function, so just undo this change